src/app/_server.js

```diff
@@ -1,59 +1,85 @@
 import React from 'react';
 import docTemplate from './HTML.js';
 import { renderToString } from 'react-dom/server';
-import { match, RouterContext } from 'react-router';
+import { StaticRouter } from 'react-router';
 import { Provider } from 'react-redux';
 import Helmet from 'react-helmet';
 import {encode} from '@tuxsudo/b64';
 import routes from './routes.js';
 import getStore from './store.js';
 import { minify } from 'html-minifier';
-import { resolve } from './hocs/ss-resolve';
+// import { resolve } from './hocs/ss-resolve';
 import {selectHTTPResponseCode} from './store.js';
 
 import * as env from './env.js';
 
 
 export default (req, res, next) => {
-    match({ routes, location: req.url }, (err, redirect, props) => {
-
-        if (err) {
-            return next(err);
-
-        } else if (redirect) {
-            res.redirect(redirect.pathname + redirect.search)
-
-        } else if (props) {
-            const store = getStore();
-
-            resolve(props, store)
```

Comment:
To update to RR4, we will need to enable the server side functionality first and so this `resolve` functionality needs to be enabled or replicated...